doc/api/vm.md

```diff
@@ -304,29 +304,6 @@ console.log(util.inspect(sandbox));
 // { globalVar: 1024 }
 ```
 
-## vm.runInDebugContext(code)
-<!-- YAML
-added: v0.11.14
--->
```

Comment:
I think the description should give a hint as to why its deprecated: perhaps `description: Deprecated because underlying v8 functionality is scheduled for removal`?